### ISSUE TITLE
Pager - accessibility violation with hiding-pager-elements

### ISSUE THREAD

**Issue by divyajindel** [Sage/carbon#4943]
### Current behaviour

Ensures ARIA attributes are allowed for an element's role
<img width="734" alt="Screenshot 2022-03-07 at 15 58 08" src="https://user-images.githubusercontent.com/88449494/157070183-e39acb0a-2814-4969-a659-cab7cc02a81d.png">

### Expected behaviour

No axe violation should occur.

### CodeSandbox or Storybook URL

https://carbon.sage.com/?path=/story/pager--hiding-pager-elements

### Carbon Version

latest

### What browsers are you seeing the problem on?

Firefox, Chrome, Safari

### What Operating System are you seeing the problem on?

MacOS

### Confidentiality

- [X] I confirm there is no confidential or commercially sensitive information included.

**Comment by edleeks87** [comment]
duplicate of #4840
